content/en/integrations/amazon_eks_fargate.md

```diff
@@ -0,0 +1,375 @@
+---
+categories:
+- cloud
+- aws
+- log collection
+ddtype: check
```

Comment:
```suggestion
```

We should remove this i think and leave it empty since this is not an Agent check at all.